docs/jwst/introduction.rst

```diff
@@ -157,7 +157,12 @@ Running From Within Python
 ==========================
 
 You can execute a pipeline or a step from within python by using the
-``call`` method of the class:
+``call`` or ``run`` methods of the class, or by calling the pipeline or
+step instance directly.
+
+The ``call`` method creates a new instance of the class and runs the pipeline or
+step. Optional parameter settings can be specified by supplying a configuration file,
+or via keyword. Examples are shown on the :ref:`Execute via call()<call_examples>` page.
```

Comment:
Can you think of a way to make it more explicit here that you're referring to keyword arguments passed to the method, as opposed to the generic way that folks immediately think of the term "keyword" (i.e. a FITS header keyword). Perhaps just "or via keyword arguments".

Reply:
I changed it to "keyword arguments", which I think makes it clear.